MANPAGE.md

```diff
@@ -140,6 +140,12 @@ this help text
 
 105: Could not convert number when parse the contents of /proc/meminfo
 
+106: Could not open /proc/spl/kstat/zfs/arcstats despite it existing
+
+107: Could read /proc/spl/kstat/zfs/arcstats
+
+108: Could not parse /proc/spl/kstat/zfs/arcstats contents
+
```

Comment:
Fatal error is too much here. Keep in mind that this depends on the behavoir of a debug file of an out-of-tree kernel module